res/values/strings.xml

```diff
@@ -461,10 +461,10 @@
     <string name="pref_auto_folder_moves">Automatic moves to DeltaChat folder</string>
     <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox</string>
     <string name="pref_email_interaction_title">Email interaction</string>
-    <string name="pref_show_emails">Show classic emails</string>
-    <string name="pref_show_emails_no">No, chats only</string>
+    <string name="pref_show_emails">Show all emails</string>
+    <string name="pref_show_emails_no">No, only chats via Delta Chat</string>
     <string name="pref_show_emails_accepted_contacts">For accepted contacts</string>
-    <string name="pref_show_emails_all">All</string>
+    <string name="pref_show_emails_all">Yes, show all emails from my mail account</string>
```

Comment:
I think this line is too long to be displayed nicely. How about

"Yes, all emails"

As we always display this together with "Show classic/all emails" it should be clear what is meant.

Reply:
Ok, I asked my brother and my mother. My brother would have understood your formulations, my mother would have preferred my formulations. 

I am still in favor of my formulation because this is an important setting and it is not good if even a few people do not understand this. Also, these are the settings, people will not look at this too often.

OTOH, of course, most users will be rather as techie as my brother.